docs/sources/index.md

```diff
@@ -293,5 +292,44 @@ TypesafeConfig.fromHoconString(jsonString, automaticDescription)
 
 
 ```
-
 Please check other ways to load the hocon file in `TypesafeConfig`
+
+## Command Line Arguments Source
+Simply source config from command line arguments
```

Comment:
Make the heading `Simple command line arguments` a subheading. That is by adding `###`

`### Simple command line arguments`

Reply:
I initially had that intent, but not confident so much. Let's make it